pkg/operator/certrotation/annotations.go

```diff
@@ -16,23 +16,30 @@ const (
 	CertificateIssuer = "auth.openshift.io/certificate-issuer"
 	// CertificateHostnames contains the hostnames used by a signer.
 	CertificateHostnames = "auth.openshift.io/certificate-hostnames"
-	// AutoRegenerateAfterOfflineExpiryAnnotation contains a link to PR and an e2e test name which verifies
+	// CertificateTestNameAnnotation is an e2e test name which verifies that TLS artifact is created and used correctly
+	CertificateTestNameAnnotation string = "certificates.openshift.io/test-name"
```

Comment:
Is it a name of an e2e test in origin repo?

Reply:
How will this annotation be used?

Reply:
This is the name of e2e test, yes. This test will be running during suspend/shutdown test in [this task](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/cert-rotation/test/openshift-e2e-cert-rotation-test-commands.sh#L13-L24)